client/src/components/Support Page/SupportPage.js

```diff
@@ -27,11 +28,20 @@ const SupportPage = () => {
   const classes = useStyles();
 
   return (
-    <Container maxWidth="sm">
-      <div style={{ marginTop: "120px" }} className={classes.root}>
-        <Tabs />
-      </div>
-    </Container>
+    <div>
+      {/* // <main style={{ height: "70vh", display: "grid", placeItems: "center" }}> */}
```

Comment:
We can remove this comment if not required

Reply:
done